code/lib/preview/src/globals/types.ts

```diff
@@ -0,0 +1,17 @@
+import type { ModuleInfo } from '@fal-works/esbuild-plugin-global-externals';
+
+// Here we map the name of a module to their NAME in the global scope.
+export enum Keys {
```

Comment:
Why is this an `enum` and not just an object?

Reply:
Do we have a preference? I just copied the code from the manager package.

Reply:
I am not a fan of enums.  Here's a short video from a friend of mine that talks about some of the problems/footguns with them.  AFAICT, the only real use for them is when setting options in a bitwise fashion.  https://www.youtube.com/watch?v=0fTdCSH_QEU&t=1s